activerecord/test/models/admin/user.rb

```diff
@@ -14,6 +14,7 @@ def load(s)
   end
 
   belongs_to :account
+  belongs_to :region
```

Comment:
using a subclass you should be able to define the association in the test-case as well.